Added test set size to (#103)
centre-for-humanities-computing/danish-foundation-models: test-dataset-plan.md

> # DadsBench: DAnish Domain-Specific language modelling Benchmark (DadsBench)

Reviewer: Small change to the acronym, to make the it more clear:

```suggestion
# DadsBench: *Da*nish *d*omain-*s*pecific language modelling *Bench*mark
```

> # DadsBench: *Da*nish *d*omain-*s*pecific language modelling *Bench*mark v.1

Reviewer: My bad, single asterisks on Github markdown is italics, apparently, with double being bold:

```suggestion
# DadsBench: **Da**nish **d**omain-**s**pecific language modelling **Bench**mark v.1
```

data from each of the following tasks:

- Danish web data (OSCAR-da)

> - Danish social media data (Twitter, General Discussions from DAGW and Reddit-da)

Reviewer: You said you wanted to split up Twitter and Reddit-da, but they're still together?

- Danish legal data (Retsinformation and Skat.dk, both from Danish Gigaword)

> - Danish clinical data (PSYCOP,  Electronic Healthcare records from psychiatri from the Central Denmark Region)

Reviewer: I don't think it's necessary to be that specific here, as we aren't with the others:

```suggestion
- Danish clinical data (PSYCOP,  Electronic psychiatric records)
```

comparable, they have all been taken from the same
[OSCAR](https://ids-pub.bsz-bw.de/frontdoor/index/index/docId/9021) dataset.

- English data ([OSCAR-en](https://huggingface.co/datasets/oscar/viewer/unshuffled_deduplicated_en/train))
- Norwegian Bokmål data ([OSCAR-nb](https://huggingface.co/datasets/oscar/viewer/unshuffled_deduplicated_no/train))
- Norwegian Nynorsk data ([OSCAR-nn](https://huggingface.co/datasets/oscar/viewer/unshuffled_deduplicated_nn/train))
- Swedish data ([OSCAR-sv](https://huggingface.co/datasets/oscar/viewer/unshuffled_deduplicated_sv/train))
- Icelandic data ([OSCAR-is](https://huggingface.co/datasets/oscar/viewer/unshuffled_deduplicated_is/train))

### The dataset:

> The DadsBench dataset consist of two main datasets. The Danish  set and a non-danish set consisting of example Non-danish set intended to model the generalizability of the model to adjacent languages.

Reviewer: Typos and phrasing:

```suggestion
The DadsBench dataset consists of two parts: Domain-specific Danish texts, as well as web data in languages similar to Danish, where the non-Danish part is intended to evaluate the model's generalisation capabilities to adjacent languages.
```

> The performance over the dataset is measures using perplexity which is calcuted pr. subset then averages across domain and then across each of the two subset resulting in two scores, one for Danish and one non-Danish languages.

Reviewer: ```suggestion
The performance over the dataset is measured using perplexity. Each part is evaluated using a weighted average of the perplexities of the individual domains within the part, and the final performance is then a weighted average of the scores of each of the two parts.
```

## Danish Test Set

| Domain | Dataset  | Open Source | Tokens |
| ------ | -------- | ---- | ------ |
| Web    | OSCAR-da | ✅   | 25 000 |
| Social media | Reddit-da,  Twitter, General Discussions  | ✅  | 25 000 |

> | Social media |  Twitter, General Discussions | (only tweet ids) | 25 000 |

Reviewer: `Twitter` and `General Discussions` appear in both of these domains?

| Legal | Retsinformation & Skat.dk | ✅ | 25 000 |
| Clinical | PSYCOP project |  | 25 000 |
| Wiki | Danish Wikipedia | ✅ | 25 000 |

> | News | DaNews |✅ | 25 000 |

Reviewer: ```suggestion
| News | DaNews | ✅ | 25 000 |
```

> | Books | Gutenberg, Danish literature, WikiBooks and WikiSource |✅ | 25 000 |

Reviewer: ```suggestion
| Books | Gutenberg, Danish literature, WikiBooks and WikiSource | ✅ | 25 000 |
```

> | spontaneous speech | OpenSubtitles and Spontaneous Speech |✅ | 25 000 |

Reviewer: ```suggestion
| Spontaneous speech | OpenSubtitles and Spontaneous Speech | ✅ | 25 000 |
```

> | Dialects | Botxt |✅ | 25 000 |

Reviewer: ```suggestion
| Dialects | Botxt | ✅ | 25 000 |
```

| Speeches | FTSpeech | ✅ | 25 000 |  
| **Sum** (sum open-souce) |  |  | 275 000 (225 000) |

> ## Non-danish Test Set

Reviewer: ```suggestion
## Non-Danish Test Set
```
